swagger/control/swagger.yaml

```diff
@@ -208,12 +174,16 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/Channel"
-      rels:
-        $ref: "#/definitions/Rels"
+      prev:
+        $ref: "#/definitions/OptionalRel"
+      next:
+        $ref: "#/definitions/OptionalRel"
     required:
       - page
+      - page_count
       - per_page
       - channels
-      - rels
+      - prev
```

Comment:
Since you already added `page_count`, I think that prev and next are now obsolete since the client needs to iterate over `page_count` and specifying the `page` parameter in `list_channels` endpoint.